app/_locales/es_419/messages.json

```diff
@@ -6,7 +6,13 @@
     "message": "Versión, centro de soporte técnico e información de contacto"
   },
   "acceleratingATransaction": {
-    "message": "* Aumentar una transacción utilizando un precio de gas más alto aumenta sus posibilidades de ser procesada más rápido por la red, pero esto no siempre está garantizado."
+    "message": "* Usar un precio de combustible más alto para acelerar una transacción aumenta las posibilidades de un procesamiento más rápido en la red, pero esto no siempre se garantiza."
```

Comment:
Previously we had decided to keep 'gas' in English as it's a bit of a technical term in Ethereum. 'Combustible' is a good translation, but is it confusing?

Reply:
i think keeping "gas" is correct